source/JsMaterialX/JsMaterialXView/src/index.js

```diff
@@ -6,13 +6,17 @@ import { EffectComposer } from 'three/examples/jsm/postprocessing/EffectComposer
 import { RenderPass } from 'three/examples/jsm/postprocessing/RenderPass.js';
 import { ShaderPass } from 'three/examples/jsm/postprocessing/ShaderPass.js';
 import { GammaCorrectionShader } from 'three/examples/jsm/shaders/GammaCorrectionShader.js';
+import { Matrix4, Uniform } from 'three';
 
-let camera, scene, model, renderer, composer, controls;
+let camera, scene, model, renderer, composer, controls, uniforms, mx;
 
 let normalMat = new THREE.Matrix3();
 let viewProjMat = new THREE.Matrix4();
 let worldViewPos = new THREE.Vector3();
 
+const fileParam = new URLSearchParams(document.location.search).get("file");
+let materialFilename = fileParam ? fileParam : 'material.mtlx'
```

Comment:
Instead of loading a default material.mtlx which currently doesn't exist I think we could just create a new document and add a standard_surface and surfacematerial node in code.

Reply:
Sounds good. Added that fallback when the programmatic part was not working